bindings/python/src/models.rs

```diff
@@ -105,6 +106,22 @@ impl PyModel {
         }
     }
 
+    fn tokenize(&self, tokens: &str) -> PyResult<Vec<PyToken>> {
```

Comment:
Why to we need this ? `encode` is enough, no ?

Reply:
These serve different purpose. `PreTokenizedString::tokenize` is deep inside the `encode` pipeline, and some use-cases require customization.